geode-web-management/src/main/java/org/apache/geode/management/internal/rest/controllers/RegionManagementController.java

```diff
@@ -38,8 +39,21 @@
       @RequestBody RegionConfig regionConfig) {
     ClusterManagementResult result =
         clusterManagementService.create(regionConfig, "cluster");
-    return new ResponseEntity<>(result,
-        result.isSuccessful() ? HttpStatus.CREATED : HttpStatus.INTERNAL_SERVER_ERROR);
+
+    HttpStatus httpStatus;
```

Comment:
If we initialize this as `INTERNAL_SERVER_ERROR` it simplifies the conditional block at line 49